sdv/constraints/base.py

```diff
@@ -200,6 +200,7 @@ def reverse_transform(self, table_data):
             pandas.DataFrame:
                 Input data unmodified.
         """
+        table_data = table_data.copy()
```

Comment:
we should do this for `transform` to no?

Reply:
Yep, sorry, I pushed that version to check if I rebased correctly. I just pushed the fixed version with the changes.